### DISCUSSION TITLE
SmartStart stopped working on Aeotec Z-Stick

### DISCUSSION THREAD

**Discussion by mazmar** [zwave-js/zwave-js-ui#2429] [Q&A]
I've upgraded my z-stick with latest fw few months ago and smartstart was working in zwavejs2mqtt. 

Today I bought new zwave termostat and found out all my provisionong entities are gone and I can' add new one with error that my controller does not support smartstart

any ideas?

It shows i have z-stick
FW 1.2 
SDK 6.7.0

I have latest 6.9.1 zwavejs2mqtt

**Comment by robertsLando** [comment]
@AlCalzone

**Reply by AlCalzone** [reply]
Please make a [driver log](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/generating-logs?id=driver-logs) of the startup, loglevel `debug` or `silly` and attach it here as a file.
